### PR TITLE
Add failing tests to action output

### PR DESCRIPTION
**Pull request by zigavehovec**
This PR extends `action-junit-report` to also include failed test names and descriptions in addition to `skipped`, `failed` and `passed` tests count. 
The output is then used to send the failing tests in a Slack message. 

A few things to note: 
- The output is limited to 2500 chars, to avoid going over Slack's section text length limit (action fails and doesn't send the message if we go over the limit)
- Since this solution is not generic and is build for a specific use-case, it isn't meant to be contributed back to the original repo. Building it in a more generic way would create a bigger complexity in our GH action flow.
- Keeping this fork up to date with the original repo should not be too much effort, since we only added 3 lines of code to the original `main.ts` file. All other changes are added as completely new files.

### CODE REVIEW

**Review by strangesource — COMMENTED** [review]
Very nice! 💯 

We probably have to update the content of the `dist` folder as well as it contains the transpiled code if I am not mistaken? 🤔

**Comment by zigavehovec** [comment]
> Very nice! 💯
> 
> We probably have to update the content of the `dist` folder as well as it contains the transpiled code if I am not mistaken? 🤔

Yes, you are right 🙂 Done ✔️
https://github.com/bitmovin-engineering/action-junit-report/pull/1/commits/7f7b43d1053d724598cd0b842be652c2bab670f0

File: README.md [hunk 1]
```diff
@@ -104,12 +104,13 @@ After action execution it will return the test counts as output.
...
-| `outputs.passed`      | The number of passed test cases.                                                       |
-| `outputs.skipped`     | The number of skipped test cases.                                                      |
-| `outputs.failed`      | The number of failed test cases.                                                       |
+| **Output**            | **Description**                                           |
+|-----------------------|-----------------------------------------------------------|
+| `outputs.total`       | The total number of test cases covered by this test-step. |
+| `outputs.passed`      | The number of passed test cases.                          |
+| `outputs.skipped`     | The number of skipped test cases.                         |
+| `outputs.failed`      | The number of failed test cases.                          |
+| `outputs.failedTests` | Names and descriptions of failed tests.                   |
```
Comment (strangesource) [inline README.md]: Nit: We could try to keep the changes minimal an only append a line to make merges smoother.

**Review by strangesource — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (zigavehovec) [reply README.md]: Good point!
https://github.com/bitmovin-engineering/action-junit-report/pull/1/commits/881b5206923bbf3ee46749f7fe044ebbe32290d6

**Review by matamegger — APPROVED** [review]
Nice! 🧊

**Pull request merged**